test/integration/transactions/transactions.spec.test.js

```diff
@@ -90,7 +90,16 @@ const SKIP_TESTS = [
 
   // TODO(NODE-2034): Will be implemented as part of NODE-2034
   'Client side error in command starting transaction',
-  'Client side error when transaction is in progress'
+  'Client side error when transaction is in progress',
+
+  // TODO(NODE-5855): Unskip Transactions Spec Unified Tests mongos-unpin.unpin
+  'unpin after TransientTransactionError error on commit',
```

Comment:
We discussed only skipping on latest, which is a smoke test anyway, these tests still pass for servers that are used in production, we can continue to make sure these work on those servers.

Reply:
Thanks for the reminder, I've changed it to skipped on latest.

Reply:
I still see the test failing in EVG:

https://spruce.mongodb.com/task/mongo_node_driver_next_windows_vsCurrent_large_gallium_test_latest_sharded_cluster_patch_8f7bb593cbd1b2cd69c76702dbd1125479d1055a_65b7f9f6850e61525b80a452_24_01_29_19_18_15/logs?execution=0&sortBy=STATUS&sortDir=ASC

Is the server version correct?

Reply:
I meant I skipped it within the code.